test/hotspot/jtreg/compiler/floatingpoint/TestSubNodeFloatDoubleNegation.java

```diff
@@ -55,7 +55,8 @@ public static void assertResults() {
     @Test
     // Match a generic SubNode, because scalar Float16 operations are often
     // performed as float operations.
-    @IR(counts = { IRNode.SUB, "2" })
+    @IR(counts = { IRNode.SUB, "2" }, applyIfPlatform = {"riscv64", "false"})
+    @IR(counts = { IRNode.SUB, ">= 2" }, applyIfPlatform = {"riscv64", "true"})
```

Comment:
Would it perhaps make sense to fix the number of `SubNode`s or does the `Float16` code add a bunch of them on RISC-V?